rust/perspective-python/perspective/widget/viewer/viewer.py

```diff
@@ -165,33 +165,53 @@ def load(self, data, **options):
         sort, etc.) is cleared to prevent applying settings on columns that
         don't exist.
 
+        When a ``perspective.AsyncTable`` is loaded, the widget's interface
+        becomes async. Methods which operate on the underlying Perspective
+        view, inclusive of the ``load()`` call itself, return coroutine values
+        which must be awaited.
```

Comment:
This presto-change-o magic API that switches between sync/async might be a little too weird. I don't know much about Python typing but it seems like it could be hard to write good types for this. Maybe mypy supports something like type state?